apps/authentication/models/connection_token.py

```diff
@@ -317,4 +333,12 @@ def expire_at(self):
         return (timezone.now() + timezone.timedelta(days=365)).timestamp()
 
     def is_valid(self):
-        return True
+        return super().is_valid()
+
+    def get_permed_account(self):
+        account = self.asset.accounts.filter(name=self.account).first()
+        if not account:
+            return None
+        account.actions = ActionChoices.all()
+        account.date_expired = timezone.now() + timezone.timedelta(days=5)
+        return account
```

Comment:
No major issues were found during analysis. No changes have been suggested in this file.